server/static/css/viewer-v2.css

```
@@ -0,0 +1,700 @@
+
+body.viewer-body-v2 {
+  /* hudTokens light palette (viewer.jsx:32–37) */
+  --color-bg-deep: #f8fafc;              /* lightBg0 — page */
+  --color-bg-base: #ffffff;              /* lightBg1 — card */
+  --color-bg-elevated: #f1f5f9;          /* lightBg2 — raised (inputs, pill) */
+  --color-text-primary: #0f172a;         /* lightText */
+  --color-text-secondary: #475569;       /* lightTextDim */
+  --color-text-bright: #0f172a;
+  --hud-line: rgba(15, 23, 42, 0.12);    /* lightLine */
+  --hud-line-strong: rgba(15, 23, 42, 0.22);
+  background: var(--color-bg-deep);
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)